pkg/models/plan_payments.go

```diff
@@ -0,0 +1,133 @@
+package models
+
+import (
+	"github.com/lib/pq"
+	"time"
+
+	"github.com/cmsgov/mint-app/pkg/models/anticipatedpaymentfrequencytype"
```

Comment:
For consistency, I think these should all be defined in the `plan_payment.go` file.  I imagine your rational here is to have small separate files. I think it is more important to stick to the established convention of the enum being in the model package, and defined in the file for the model they represent. Also gql is configured to match types only to the models package, if you define an enum in a separate package, GQL generates a different object in the model package, and doesn't automatically resolve the type. (I'll point to this in schema resolvers)

I'm open to a conversation about defining in a separate package / file, but I think that should be a separate Story to refactor all existing data so we don't need to support multiple paradigms.

Reply:
The intention was primarily to allow scoping, similar to C# where you can reference the package and the linter knows what to suggest. That said, I think you're right that convention is more important at this stage.